docs/guide/content_model.md

```diff
@@ -125,6 +125,10 @@ They determine what Fields of what Field Types will be included in all Content i
 
     You can assign each Field defined in a Content Type to a group by selecting one of the groups in the Category drop-down. [Available groups can be configured in the content repository](configuration.md#content-repository-configuration).
 
+!!! caution
+    
+    In case of Content Types containing many Field Types you should be aware of possible memory-related issues on publishing/editing. They are caused by the limitation of how many `$_POST` input variables may be accepted. The easiest way to fix them is increasing `max_input_vars` value in `php.ini` configuration file.
```

Comment:
```suggestion
    In case of Content Types containing many Field Types you should be aware of possible memory-related issues on publishing/editing. They are caused by the limitation of how many `$_POST` input variables may be accepted. The easiest way to fix them is increasing `max_input_vars` value in `php.ini` configuration file. You may also experience performance problems with such large Content Types, in particular when you have many Content items.
```
Should we define a threshold, beyond which you should consider doing things differently? 20? 30?

Reply:
Not sure about this one, haven't tested any number in particular. :/

Reply:
Yeah, I didn't mean a scientifically valdi limit, just a fencepost-ish thing, saying something like "beyond this you are entering uncharted territory, we don't intend our software to be used like this and haven't tested it, perhaps you should consider other ways of doing what you need to do".
CC @ezsystems/documentation-team